icons/nim/nim-line-wordmark.svg

```diff
@@ -0,0 +1 @@
vg" viewBox="0 0 120 120"><path fill="#ffe953" d="m86.886 42.81-.733.522-3.931 2.81v2.304h4.664zm-61.941.292-.307.244s-1.772 1.397-3.568 2.783c-1.961-.038-5.289.35-7.407 1.063-1.747-1.127-3.329-2.362-3.329-2.362l-.419-.334-.27.463s-1.355 2.327-2.233 3.742c-1.192.64-2.388 1.387-3.496 2.321a263.1 263.1 0 0 1-2.768-1.13L0 49.405l.554 1.12c1.787 3.604 3.024 7.363 6.478 9.656l.394.263.253-.402c1.225-1.935 3.634-3.408 6.685-4.367 3.05-.959 6.72-1.405 10.396-1.36 3.675.044 7.356.58 10.426 1.557 3.07.977 5.516 2.406 6.762 4.145l.236.328.355-.187c3.664-1.91 5.036-5.95 6.395-9.533l.346-.91-.923.306c-.155.051-1.784.604-2.987 1.052-.824-.813-2.159-1.782-3.086-2.35-.915-1.67-2.202-3.86-2.202-3.86l-.26-.444-.415.307s-1.52 1.126-3.23 2.329c-2.362-.436-5.166-.932-7.647-.832-1.638-1.356-3.294-2.856-3.294-2.856zm-.035 1.207c.26.235 1.57 1.432 3.14 2.723l.139.114.18-.01c2.417-.132 5.374.408 7.836.862l.191.037.159-.112c1.602-1.122 2.658-1.909 2.983-2.15.246.42 1.177 2.003 1.98 3.474l.06.11.106.066c.925.558 2.639 1.79 3.271 2.48l.212.231.294-.112c.816-.311 1.522-.542 2.196-.772-1.212 3.18-2.526 6.311-5.315 7.91-1.481-1.822-3.886-3.22-6.874-4.172-3.18-1.012-6.94-1.554-10.697-1.6-3.757-.045-7.511.406-10.685 1.403-2.966.933-5.349 2.398-6.805 4.385-2.61-1.883-3.874-4.832-5.359-7.968.595.246 1.018.425 1.87.764l.264.106.215-.189c1.13-1 2.41-1.766 3.705-2.455l.11-.058.066-.106c.79-1.27 1.776-2.967 2.036-3.412.334.263 1.457 1.161 3.124 2.218l.19.121.215-.077c1.912-.695 5.693-1.135 7.515-1.077l.166.004.131-.1a341.672 341.672 0 0 0 3.381-2.638zm61.05.3v2.911h-2.812v-.901zm-26.705 7.718v21.531l4.664 3.333v-20.2h5.412c1.367 0 2.21.984 2.21 2.208v14.885h4.664V58.398c0-3.218-2.854-6.071-6.072-6.071zm22.967 0v21.757h4.664V52.327Zm10.681 0v21.757h4.665V56.99h6.552v17.093h4.664V56.99h4.342c.698 0 1.237.225 1.611.599.375.373.599.911.599 1.61v17.99l4.664-3.33v-15.46c0-3.436-2.635-6.071-6.071-6.071zm-32.722.926h9.952c2.657 0 5.145 2.488 5.145 5.145v14.76h-2.811v-13.96c0-1.68-1.3-3.134-3.136-3.134h-6.338V75.39l-2.812-2.009zm22.967 0h2.812v19.904h-2.812zm10.682 0h20.099c2.973 0 5.145 2.172 5.145 5.145v14.983l-2.812 2.01V59.198c0-.904-.31-1.703-.87-2.264-.562-.56-1.362-.87-2.266-.87h-5.268v17.093h-2.812V56.064H96.64v17.093h-2.81zm-47.25 4.483c-2.075 3.074-5.515 5.25-7.644 6.42-.668.367-1.924.715-2.982.943-.982.211-1.677.31-1.788.326L24.82 60.58l-9.402 4.747a32.638 32.638 0 0 1-1.773-.34c-1.052-.228-2.304-.556-3.017-.887-3.122-1.669-5.2-3.618-7.435-6.34l-.786.471L6.743 68.73l.023.056.035.047c3.875 5.113 10.699 7.804 17.61 7.916 6.91.112 13.953-2.36 18.255-7.714l.039-.046.023-.056c1.986-4.475 4.663-10.757 4.663-10.757zM4.209 60.17c1.739 1.843 3.553 3.452 5.994 4.755l.012.006.011.004c.867.405 2.14.724 3.223.959 1.082.235 1.965.376 1.965.376l.148.023 9.254-4.67 9.197 4.765.147-.022s.897-.127 1.99-.362c1.091-.236 2.369-.562 3.232-1.035 1.59-.874 3.909-2.307 5.913-4.288-.863 2.016-1.934 4.543-3.38 7.802-4.081 5.056-10.824 7.448-17.49 7.34-6.664-.109-13.199-2.72-16.856-7.52l-3.36-8.133Z" /></svg>
```

Comment:
looks like the viewbox is not 128x128 here

Reply:
Great catch!

Reply:
I wonder why check-bot didn't catch this error btw

Reply:
I was just lucky 😁 but gave me the idea to check out github actions for such simple checks like is there a `viewBox `attribute with `0 0 128 128` or similar. I will do some experiments, on how to automate the pure xml checks in svg.

Reply:
@weh It would be great if you could take a look!

We already have a script that should do that, but it seems like it's not working here for some reason.
This is the part that should have caught the error: https://github.com/devicons/devicon/blob/master/.github/scripts/check_icon_pr.py#L146
It's ran by the [check-icon-pr](https://github.com/devicons/devicon/tree/master/.github/workflows#L18C10-L18C10) workflow